stable/chronograf/values.yaml

```diff
@@ -29,10 +34,23 @@ resources:
     
 ## Configure the ingress object to hook into existing infastructure
 ## ref : http://kubernetes.io/docs/user-guide/ingress/
+## OPTIONALLY you can set .Values.ingress.secretName to set which secret to use    
 ## 
 ingress:
   enabled: false
   tls: false
   hostname: chronograf.foobar.com
   annotations:
-    kubernetes.io/ingress.class: "nginx"
\ No newline at end of file
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
I would leave this annotation off by default so that the cluster default is used out of the box